src/Transactions/TransactionDetails/TransactionDetailsContainer.js

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { withRouter } from 'react-router-dom';
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  LoadingPane,
+} from '@folio/stripes-acq-components';
+
+import {
+  fiscalYearResource,
+  fundsResource,
+  transactionsResource,
+  transactionByUrlIdResource,
+} from '../../common/resources';
+import { FISCAL_YEARS_API } from '../../common/const';
+import TransactionDetails from './TransactionDetails';
+
+const TransactionDetailsContainer = ({
+  history,
+  match,
+  mutator,
+  resources,
+}) => {
+  const transactionId = match.params.id;
+  const transaction = get(resources, ['transactionDetails', 'records', '0']);
+  const onClose = history.goBack;
```

Comment:
is it correct when we open pane with direct link?

Reply:
absolutely not

Reply:
I think it should be fixed when transactions list route is implemneted